Library/Formula/libvpx.rb

```diff
@@ -2,8 +2,10 @@
 
 class Libvpx < Formula
   homepage 'http://www.webmproject.org/code/'
-  url 'https://webm.googlecode.com/files/libvpx-v1.3.0.tar.bz2'
-  sha1 '191b95817aede8c136cc3f3745fb1b8c50e6d5dc'
+  url "https://chromium.googlesource.com/webm/libvpx",
+    :using => :git,
+    :revision => "c74bf6d889992c3cabe017ec353ca85c323107cd"
+  version "1.4.0"
```

Comment:
We can use the tarball from its official Github mirror: https://github.com/webmproject/libvpx/releases